packages/snap-preact-components/src/components/Molecules/FacetHierarchyOptions/FacetHierarchyOptions.stories.tsx

```diff
@@ -52,6 +52,26 @@ export default {
 			},
 			control: { type: 'boolean' },
 		},
+		previewOnFocus: {
```

Comment:
Nothing to do with your work here... But looking at this now, I wonder if we should refactor these to be 'onValueFocus' and pass the `value.preview()` in an anonymous function. Seems like it would be a better way to handle the autocomplete focus usage (as the `preview` method is only present in an AutocompleteStore value).

Reply:
Wait... couldn't it just be thrown into valueProps???